src/app/components/list/filters/index.html

```diff
@@ -69,48 +67,66 @@
     </sky-demo-page-property>
   </sky-demo-page-properties>
 
   <sky-demo-page-properties sectionHeading="ListFilterModel properties">
     <sky-demo-page-property
       propertyName="name"
     >
-    The name for the filter.
+    Specifies the name of the filter. This property accepts <stache-code>string</stache-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
-      propertyName="filter"
+      propertyName="defaultValue"
       isOptional="true"
     >
-    Required when using in memory data provider. This is the function applied to determine whether an item is filtered. Acceptable values: <stache-code>(item: ListItemModel, filterValue: any) => boolean</stache-code>.
+    Specifies the default value of the filter. If the value of the filter is set to the default value, then the filter is not applied.
     </sky-demo-page-property>
     <sky-demo-page-property
-      propertyName="value"
+      propertyName="dismissible"
       isOptional="true"
+      defaultValue="true"
     >
-    The value of the filter.
+    Indicates whether users can dismiss the filter's summary item. This property accepts <stache-code>boolean</stache-code> values.
     </sky-demo-page-property>
-
     <sky-demo-page-property
-      propertyName="defaultValue"
+      propertyName="filterFunction"
       isOptional="true"
     >
-    The default value of the filter. If the value of the filter is set to the default value, then the filter is not applied.
+    Specifies the function to apply to determine whether an item is filtered. This property is required when using in memory data provider. Acceptable values: <stache-code>(item: ListItemModel, filterValue: any) => boolean</stache-code>.
     </sky-demo-page-property>
     <sky-demo-page-property
-      propertyName="dismissible"
+      propertyName="label"
       isOptional="true"
-      defaultValue="true"
     >
-    Determines whether the filter's summary item can be dismissed.
+    Specifies the text for the filter's summary item. If not specified, then the filter summary item text defaults to the <stache-code>value</stache-code> of the filter. This property accepts <stache-code>string</stache-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
-      propertyName="label"
+      propertyName="value"
       isOptional="true"
-      defaultValue="true"
     >
-    Specifies the text for the filter's summary item. If not specified, then the filter summary item text defaults to the <stache-code>value</stache-code> of the filter.
+    Specifies the value of the filter.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="ListItemModel properties">
+    <sky-demo-page-property
+      propertyName="data"
+    >
+    Specifies the data for the item in the list.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="id"
```

Comment:
Should we indicate that this property is required? Based on https://github.com/blackbaud/skyux-list-builder-common/blob/master/src/app/public/state/items/item.model.ts#L8, I think it might be required.

Reply:
Yes. Good catch.